js/persistency-sequence-generate-topic/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "persistecy-sequence-generate-topic",
```

Comment:
not sure if it shouldn't be named "@scramjet/persistecy-sequence-generate-topic"